### ISSUE TITLE
Calypsoify: gulp task created files

### ISSUE THREAD

**Issue by jeherve** [Automattic/jetpack#10626]
When running `gulp sass:calypsoify`, the following files are created:

```
	modules/calypsoify/style-gutenberg-min.rtl.rtl.css
	modules/calypsoify/style-gutenberg-min.rtl.rtl.css.map
	modules/calypsoify/style-gutenberg-rtl.min.rtl.css
	modules/calypsoify/style-gutenberg-rtl.min.rtl.css.map
	modules/calypsoify/style-gutenberg-rtl.rtl.css
	modules/calypsoify/style-gutenberg-rtl.rtl.css.map
	modules/calypsoify/style-gutenberg.min.rtl.css
	modules/calypsoify/style-gutenberg.min.rtl.css.map
	modules/calypsoify/style-min.rtl.rtl.css
	modules/calypsoify/style-min.rtl.rtl.css.map
	modules/calypsoify/style-rtl.min.rtl.css
	modules/calypsoify/style-rtl.min.rtl.css.map
	modules/calypsoify/style-rtl.rtl.css
	modules/calypsoify/style-rtl.rtl.css.map
	modules/calypsoify/style.min.rtl.css
	modules/calypsoify/style.min.rtl.css.map
```

It would be good to:

- Review that list since the double rtl seem a bit odd here.
- Update `.gitignore` and `.svnignore` accordingly, so the files don't come in the way when working on other things.

Related: #10371

**Comment by georgestephanis** [comment]
Fixed via #10631

**Comment by jeherve** [comment]
Reopening this. @georgestephanis Could you update `.svnignore` as well, so we don't ship .map and .scss files with the plugin?

Thank you.
